### DISCUSSION TITLE
Community contributions determination

### DISCUSSION THREAD

**Discussion by atg-abhishek** [Green-Software-Foundation/sci#90] [General]
More info @jawache  - http://bit.ly/across-workshop-community-pitch - This is related to a bigger issue, around how to identify contributors - understanding explicitly that source control is not the full picture of open source, as it excludes governance, decision making, and consensus as you mentioned. https://citation-file-format.github.io/ - this file is one approach to recognize contributors without requiring people to update the repo

_Originally posted by @Sealjay in https://github.com/Green-Software-Foundation/software_carbon_intensity/issues/84#issuecomment-925830362_

**Comment by atg-abhishek** [comment]
@Henry-WattTime @jawache @Sealjay @SaraEmilyBergman @vaughanknight would love to hear thoughts on the comment above to see where we want to head with this. Also, please tag any others in this discussion who might be able to provide some insights, thanks!

**Reply by SaraEmilyBergman** [reply]
Great topic! For context, are we specifically talking about the standard now? Or the entire foundation? if the latter, I assume many people in the Trademark group have some great ideas here (I don't know who to tag though).

For the first (e.g. scoped to the standard), I think this is a very important topic. I did like the first link you send Chris, should we have a workshop to follow the process outline there maybe?

Either way, my thinking here is that it is better to go wide than narrow here. Either we put no names, it is the entirety of the foundation behind it. Or we admit the name of anyone who has commented or written a conversation, issue or PR in the repo or has attended a Standards WG meeting. Very simplistic view of it and members will always have different levels of contributions of course. But in my view of the world, all contributions are important, including listening in to a discussion and agreeing to it, it lends weight to what we are doing and it means we have grounded in it in a wide circle of perspectives. Does that make sense?

**Reply by Sealjay** [reply]
I'm in a similar vein. I'm wondering if we want to put no names in the specification itself to ensure that the work is unbiased and stands on its own merits, rather than having a link to people and specific organisations.

Credit and attribution for a community are important, though - both personally and professionally, so would we want to have a contributors recognition at the overall foundation level? As @SaraEmilyBergman said, all contributions are important - if so, that goes outside the bounds of the standards WG, for example, but could be a more equitable way to recognise the work of a broader group of people.

**Reply by atg-abhishek** [reply]
These are both great points, I'm leaning towards having no names in that case and then having something that we can do at the GSF level maybe to at least professionally recognize the contributions in case that is helpful to those for whom this relates to their primary work as well?

**Reply by SaraEmilyBergman** [reply]
I think that sounds good, having a way of showcasing that you have contributed in a validated manner is probably something everyone in the foundation would be interested in.

**Reply by jawache** [reply]
I would prefer no names since no names implicitly imply it has the support of all current and future members. 

We can argue that if you are a member of the Foundation, you support **everything** we do because of the way we are structured with consensus and voting and consistency reviews, everyone has a voice and an opportunity to comment or object. It's like a "speak now or forever hold your peace" statement. If you don't agree with what we are doing, object, if you're objection isn't resolved to your satisfaction and what we publish is completely incompatible with your viewpoint then you have the option to leave the foundation. Our strength is in our unified front, we are all in!

I worry that if we start putting specific people's names on official outputs it breaks that implicit agreement and people can then start doing a pick and mix, saying they don't agree with the SCI because their names were not on the list but they do agree with X.

To give people credit I was thinking perhaps we can write an article. Something where we interview anyone who wants to be included, interview them about the SCI, what it means to them? Publish it on the blog simultaneously with the publication of the SCI. That's the kind of material journalists quote from when they write their articles about the SCI.

**Reply by atg-abhishek** [reply]
100% makes sense with our operating processes! I've created #109 to reflect this.
